### DISCUSSION TITLE
关于源字体 woff2 格式

### DISCUSSION THREAD

**Discussion by mobeicanyue** [KonghaYao/cn-font-split#66] [General]
1. 您在文档中说，FontPath 推荐使用 otf、ttf 字体。那我想请问如果原字体提供了 woff2 格式呢？是不是直接使用会更好 少一层转换。还是说会先 woff2 -> otf -> woff2？
2. 请问一下切割字体的时候该如何关闭 log，我尝试写 mjs 将其 log 赋值空函数，但是仍有输出。
3. 如果是 bun i -g 安装的工具，命令行可以直接运行。但是 bun i 安装，再运行 mjs 似乎就不行了，会报依赖的错。（bun 用着确实快，分字体用时4min，node 用时5min）

**Comment by KonghaYao** [comment]
1. woff2 会先转换为  otf，因为大部分工具都只能对 opentype 进行操作。而且 woff2 转换可能会有字符损失，设计师一般都会给 otf 或者 ttf 的，通用性比较高
2. log 是一个副作用，logger 的 setting 才是配置 tslog 的
```diff
fontSplit({
    destFold: './temp/node',
    FontPath: '../demo/public/SmileySans-Oblique.ttf',
    targetType: 'woff2',
    previewImage: {},
    chunkSize: 70 * 1024,
    testHTML: false,
    reporter: false,
    css: {
    },
+   logger: {
+        settings: {
+           minLevel: 4
+        }
+    }
});

```

3. bun i -g 安装到全局会自动注册 cn-font-split，如果是项目内使用 bun i , 那应该使用 ` import {fontSplit} from 'cn-font-split'; ` 在代码里面使用, 或者 bunx cn-font-split 这样命令行使用 (maybe)
